### DISCUSSION TITLE
Running Wine on desktop-less environnement

### DISCUSSION THREAD

**Discussion by actraiser30** [ptitSeb/box86#866] [General]
Hello there

I installed box86/wine in a raspberry pi 4

Just a question wine does not run always with the first try,sometimes it runs with the first time sometimes it needs to run it 3,4 times to work,is this normal?

**Comment by ptitSeb** [comment]
Well, no, wine should work on all launch.

**Comment by actraiser30** [comment]
I mean i launch it,it crashes I launch it again it works.i don't use desktop,so I use xinit with a script to run games,is there a specific launch command/s that works 100% ??

Also to run d3d games I use the --opengl command or there are more commands ??

**Comment by ptitSeb** [comment]
I always use desktop on all my machine, so I don't know if it can interfer with normal process.
Maybe I should convert this ticket as discussion to allow other person with similar setup to interact?

**Comment by actraiser30** [comment]
Yes please thank you

**Reply by ptitSeb** [reply]
Done. I also change the title to something more specific
